src/infrastructure/validation/testing/TestCase.java

```diff
@@ -0,0 +1,67 @@
+/**
+ * @author			Abhishek Saran
+ * module			Infrastructure module 
+ * team				Test Harness
+ * description		Abstract class which implements ITest Interface
+ * summary			Modules have to extend the abstract class to implement the run method for each test case  
+ */
+
+package infrastructure.validation.testing;
+
+public abstract class TestCase implements ITest {
+
+	//data members
+	/**
+	 * @param category		The category of a test case is its module name 
+	 * @param error			The error occured in the test case 
+	 * @param description	A small description about what this test case is about
+	 * @param priority		The priority level of the test case - 0 being lowest and 2 being highest
+	 */
+	private String category;
+	private String error;
+	private String description;
+	private int priority;
+
+	//methods
+	/**
+	 * abstract method to be implemented for each TestCase
+	 * @return			<code>true</code> if the test case pass 
+	 *                  and actual output is same as expected output; 
```

Comment:
Align these and make sure the when the description for the block tags go to the next line, they should be indented 4 spaces from the @. Check the alignment and indentation in other places too.